addons/apecloud-mysql/scripts/etcd-post-start.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
-endpoints=${ETCD_SERVER:-'127.0.0.1:2379'}
+if [ -n "$LOCAL_ETCD_POD_LIST" ]; then
+  IFS=',' read -ra ETCD_POD_ARRAY <<< "$LOCAL_ETCD_POD_LIST"
+  endpoints=""
+  for pod in "${ETCD_POD_ARRAY[@]}"; do
+    endpoints+="${pod}.${LOCAL_ETCD_HEADLESS}.${KB_NAMESPACE}.svc.cluster.local:${LOCAL_ETCD_PORT},"
```

Comment:
".cluster.local" should be replaced by {{ .Values.clusterDomain }}

Reply:
<img width="655" alt="截屏2024-07-23 10 21 23" src="https://github.com/user-attachments/assets/a5c28888-e2f7-4314-975b-7010b3f767f2">
Do you mean adding an environment variable and then referencing it in the script to avoid hardcoding?

Reply:
yeah, this is a solution